packages/mui/src/date-time-range-pickers/wrapper.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, Grid } from '@mui/material';
+import { useLocalizationContext } from '@mui/x-date-pickers/internals/hooks/useUtils';
+import {
+  ThemeColorSwapper,
+  ThemeColorSwapperProps,
+  useMultiLocalizationContext,
+} from '@my-sahab/mui';
+import { endOfDay, startOfDay } from 'date-fns-jalali';
+import * as React from 'react';
+import { DatePicker, DateTimePicker, TimePicker } from 'src/date-time-pickers';
+import { PickerTypes, WrappedPickerProps } from 'src/date-time-pickers/wrapper';
+import { TimeRange } from 'src/date-time-utils';
+import { Styles } from '../react-types';
+import { makeStyles } from '../tss-mui';
+
+type PickerProps<P extends PickerTypes, In, Out> = Omit<
+  WrappedPickerProps<P, In, Out>,
+  'multiLocale' | 'value' | 'onChange'
+>;
+
+type Props<P extends PickerTypes, In, Out> = {
+  value: TimeRange;
+  onChange: (value: TimeRange) => void;
+  /** shared props between two pickers */
+  pickerProps?: PickerProps<P, In, Out>;
+  fromPickerProps?: PickerProps<P, In, Out>;
+  toPickerProps?: PickerProps<P, In, Out>;
+  color?: ThemeColorSwapperProps['color'];
+} & (P extends 'TIME' ? unknown : { multiLocale?: boolean }) &
+  StyleProps;
+
+export { Props as WrappedRangePickerProps };
+
+const pickers: Record<PickerTypes, React.ElementType> = {
+  TIME: TimePicker,
+  DATE: DatePicker,
+  DATETIME: DateTimePicker,
+};
```

Comment:
share this object with `date-time-pickers/wrapper.tsx`

Reply:
They are different!
for `date-time-pickers`, the pickers are MUI pickers.
but for `range-pickers`, they are `date-time-pickers`.